### PR TITLE
Featured rating fixes

### LINKED ISSUE #108: Make mentors/users on featured page clickable and link to profile page

### PR DESCRIPTION
**Pull request by rmjordas**
**Did**:

- [x] Fix hover behavior
- [x] Fix Featured item to be a clickable link to user/mentor profile
- [x] Render `Rating` only when value is not zero

**TODO**:

- [x] Fix path for featured item

Closes #108

### CODE REVIEW

File: src/components/shared/featured.tsx [hunk 1]
```diff
@@ -57,14 +60,16 @@ const FeaturedTitle = styled.p`
...
     return (
-      <StyledFeatured>
-        <StyledImg src={this.props.src} />
-        <FeaturedContent>
-          <FeaturedName>{this.props.name}</FeaturedName>
-          <FeaturedTitle>{this.props.title}</FeaturedTitle>
-          <Rating maxValue={5} value={this.props.rating} fill="secondary" />
-        </FeaturedContent>
-      </StyledFeatured>
+      <Link to={`/${USER}/${this.props.id}`}>
```
Comment (roymoran) [inline src/components/shared/featured.tsx]: @rmjordas is this current a placeholder?

File: src/components/shared/featured.tsx [hunk 2]
```diff
@@ -57,14 +60,16 @@ const FeaturedTitle = styled.p`
...
+        <StyledFeatured>
+          <StyledImg src={this.props.src} />
+          <FeaturedContent>
+            <FeaturedName>{this.props.name}</FeaturedName>
+            <FeaturedTitle>{this.props.title}</FeaturedTitle>
+            {this.props.rating > 0 && <Rating maxValue={5} value={this.props.rating} fill="secondary" />}
```
Comment (roymoran) [inline src/components/shared/featured.tsx]: One suggestion, if the user/mentor has no rating I'm thinking we should default to 5 stars. What do you think?

I think it's better that we give people the benefit of being good mentors until they get their first rating.

Comment (rmjordas) [reply src/components/shared/featured.tsx]: Well, that might appear misleading? Maybe, we can add a badge that says they are new mentors or something. Or maybe I didn't understand?

Comment (roymoran) [reply src/components/shared/featured.tsx]: Okay that makes sense. lets leave it how it is.

File: src/components/shared/featured.tsx (hunk 1 above)
Comment (rmjordas) [reply src/components/shared/featured.tsx]: I pushed a fix

**Pull request merged**